hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/s3a/WriteOperationHelper.java

```diff
@@ -585,7 +589,8 @@ public BulkOperationState initiateOperation(final Path path,
   @Retries.RetryTranslated
   public UploadPartResult uploadPart(UploadPartRequest request)
       throws IOException {
-    return retry("upload part",
+    return retry("upload part #" + request.getPartNumber()
+        + " upload "+ request.getUploadId(),
```

Comment:
nit: s/upload/upload ID/

I was thinking of consistent log keywords so taht for any retry log we can search "upload ID" or "commit ID"

Reply:
This is the S3 multipart upload ID, so I'll use upload ID for it...its also used in BlockOutputStream